packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/RelationMultiple/index.js

```diff
@@ -67,7 +67,7 @@ const RelationMultiple = ({ fieldSchema, metadatas, name, entityId, value, conte
       staleTime: 0,
       select: (data) => ({
         ...data,
-        results: data.results.reverse(),
+        results: [...data.results].reverse(),
```

Comment:
Think there could be a unit test for this?

Reply:
Thanks, unit test has been added now. Kindly take a look.